src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -567,6 +567,11 @@ protected View getWelcomeView() {
         return new MultiBranchProjectEmptyView(this);
     }
 
+    @Override
+    public View getView(String name) {
+        return getPrimaryView();
```

Comment:
You can. There are problems ([JENKINS-32178](https://issues.jenkins-ci.org/browse/JENKINS-32178)), but this mistake is making it worse.